.github/workflows/scout.md

```diff
@@ -0,0 +1,128 @@
+---
+on:
+  command:
```

Comment:
add support for a workflow_dispatch + input topic string way to launch the workflow. Make sure to inline the input in the prompt.